From 25d335b22f732e9f46b95371a2e79b172a6c7da3 Mon Sep 17 00:00:00 2001
From: zhang sanshan <b51434@freescale.com>
Date: Mon, 17 Aug 2015 16:04:54 +0800
Subject: [PATCH 13/22] MA-6996 Allow permissive in both eng and user mode

Allow setting selinux mode to permissive in user mode building.

Signed-off-by: zhang sanshan <b51434@freescale.com>

Conflicts:
	init/Android.mk
---
 init/Android.mk | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/init/Android.mk b/init/Android.mk
index 0fbfbec..cbe099c 100644
--- a/init/Android.mk
+++ b/init/Android.mk
@@ -4,11 +4,7 @@ LOCAL_PATH:= $(call my-dir)
 
 # --
 
-ifneq (,$(filter userdebug eng,$(TARGET_BUILD_VARIANT)))
 init_options += -DALLOW_LOCAL_PROP_OVERRIDE=1 -DALLOW_DISABLE_SELINUX=1
-else
-init_options += -DALLOW_LOCAL_PROP_OVERRIDE=0 -DALLOW_DISABLE_SELINUX=0
-endif
 LOCAL_CFLAGS +=  -DALLOW_DISABLE_SELINUX=1
 
 init_options += -DLOG_UEVENTS=0
-- 
1.8.0

